ci/scripts/regress-test.sh

```diff
@@ -27,7 +27,17 @@ cargo make pre-start-playground
 cargo make link-all-in-one-binaries
 
 echo "--- Postgres regress test"
-
+apt-get update -yy
+apt-get -y install locales
```

Comment:
Is it a must to install on our environment? Or it is just a requirement for postgres?

Reply:
It is a must otherwise `psql` would throw warnings and warnings would go into the output file, which makes comparison always fail.